.github/workflows/ci.yml

```diff
@@ -13,4 +13,15 @@ jobs:
         - name: Install modules
           uses: bahmutov/npm-install@v1
         - name: Run cSpell
-          run: yarn spellcheck
\ No newline at end of file
+          run: yarn spellcheck
+    UnitTest:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/setup-node@v3
+          with:
+            node-version: 16
+        - uses: actions/checkout@v2
+        - name: Install modules
+          uses: bahmutov/npm-install@v1
+        - name: Run unit tests
+          run: yarn test
```

Comment:
can you ensure a time limit on the server (let's say 20s)?

Reply:
the smallest is 1 minute. I don't think it's necessary as it's running super fast right now

Reply:
do you think it makes sense to add it so we never run out of credits?

Reply:
for open source it's all free.